packages/components/src/navigator/navigator-screen/component.tsx

```diff
@@ -149,73 +133,11 @@ function UnconnectedNavigatorScreen(
 
 	const mergedWrapperRef = useMergeRefs( [ forwardedRef, wrapperRef ] );
 
-	if ( ! isMatch ) {
-		return null;
-	}
-
-	if ( prefersReducedMotion ) {
-		return (
-			<View
-				ref={ mergedWrapperRef }
-				className={ classes }
-				{ ...otherProps }
-			>
-				{ children }
-			</View>
-		);
-	}
-
-	const animate = {
-		opacity: 1,
-		transition: {
-			delay: animationEnterDelay,
-			duration: animationEnterDuration,
-			ease: 'easeInOut',
-		},
-		x: 0,
-	};
-	// Disable the initial animation if the screen is the very first screen to be
-	// rendered within the current `NavigatorProvider`.
-	const initial =
-		location.isInitial && ! location.isBack
-			? false
-			: {
-					opacity: 0,
-					x:
-						( isRTL() && location.isBack ) ||
-						( ! isRTL() && ! location.isBack )
-							? 50
-							: -50,
-			  };
-	const exit = {
-		delay: animationExitDelay,
-		opacity: 0,
-		x:
-			( ! isRTL() && location.isBack ) || ( isRTL() && ! location.isBack )
-				? 50
-				: -50,
-		transition: {
-			duration: animationExitDuration,
-			ease: 'easeInOut',
-		},
-	};
-
-	const animatedProps = {
-		animate,
-		exit,
-		initial,
-	};
-
-	return (
-		<motion.div
-			ref={ mergedWrapperRef }
-			className={ classes }
-			{ ...otherProps }
-			{ ...animatedProps }
-		>
+	return isMatch ? (
+		<View ref={ mergedWrapperRef } className={ classes } { ...otherProps }>
```

Comment:
Since the animations will now be processed with CSS, the test related to verifying animations run will fail. I think it's safe to delete it.

Reply:
It's a pity that we can't easily test that the initial screen doesn't animate forward, but it's a current limitation of `jsdom` 🤷‍♂️ 

https://github.com/WordPress/gutenberg/pull/56909/commits/f84568c9c229a244fa80cbc944f2937ca7ccfdf5